jme3-codepalette/src/com/jme3/gde/codepalette/layer.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE filesystem PUBLIC "-//NetBeans//DTD Filesystem 1.2//EN" "http://www.netbeans.org/dtds/filesystem-1_2.dtd">
```

Comment:
Unwanted merge issue?

Reply:
Same for the .keep?

Reply:
It would seem so. I did have some issues with the branch, previously.